test/libsolidity/smtCheckerTests/special/tx_data_immutable_fail.sol

```diff
@@ -30,7 +32,8 @@ contract C {
 
 		assert(bhash == blockhash(122));
 		assert(coin != block.coinbase);
-		assert(dif != block.difficulty);
+		assert(dif == block.difficulty);
+		assert(prevrandao != block.prevrandao);
```

Comment:
Is this correct? Shouldn't both be `!=`?

Reply:
Yes, it is wrong. Thanks.